aquadoggo/src/graphql/replication/aliased_author.rs

```diff
@@ -0,0 +1,13 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+
+use async_graphql::*;
+
+use super::public_key::PublicKey;
+
+#[derive(Debug, InputObject, SimpleObject)]
+pub struct AliasedAuthor {
```

Comment:
I'm not sure if I understand the concept of `AliasedAuthor`, could you maybe add a docstring here explaining it?

Reply:
Added an explanation. Let me know if it's helpful.

Reply:
I'm still not sure what's gonna happen with the aliased authors. I think I need to re-think a few things around it. But I don't want to get bogged down in it either. I expect it will change.